tests/test_baselines.py

```diff
@@ -33,30 +35,35 @@ def test_baselines_bimodal_linear_regression():
     X_train = X[:n]
     y_train = y[:n]
 
-    X_test = X[n:]
-    y_test = y[n:]
+    X_test = X[-100:]
+    y_test = y[-100:]
 
     model = NNffuser(
         verbose=1,
         n_repeats=20,
-        n_epochs=1000,
+        card_like=card_like,
         sde_name="vesde",
-        batch_size=64,
-        hidden_size=256,
-        learning_rate=0.001,
+        batch_size=256,
+        n_layers=3,
+        hidden_size=100,
+        learning_rate=0.005,
+        decay=0.0,
```

Comment:
idk how decay works, but the default is 0.9999 and now it is set to 0? is this coherent?

Reply:
Yeah, it is only because this is faster and I was trying to make the test run as fast as possible. Both work.